src/core/config/misc.h

```diff
@@ -141,6 +141,26 @@
 #define OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE
+ *
+ * Define to 1 to enable multipan RCP support.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE
+#define OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE 0
+#endif
+
+/**
+ * @def SPINEL_HEADER_IID_MULTIPAN_BROADCAST
+ *
+ * Define broadcast IID for spinel frames dedicated to all hosts in multipan configuration.
+ *
+ */
+#ifndef SPINEL_HEADER_IID_MULTIPAN_BROADCAST
+#define SPINEL_HEADER_IID_MULTIPAN_BROADCAST SPINEL_HEADER_IID_3
```

Comment:
Shouldn't this be something with `OPENTHREAD_CONFIG` prefix like other options? OR
There is `src/lib/spinel/openthread-spinel-config.h` file if that is the correct place for this config.